tests/basic-test.sh

```diff
@@ -41,6 +41,7 @@ fi
 validate_checkout_basic() {
     (cd $1;
      assert_has_file firstfile
+     assert_not_streq $(stat -c '%h' firstfile) 1
```

Comment:
Interesting, this is failing in `test-basic-user.sh`. Surprisingly, not in the `-U -H` case, but just in the vanilla `$OSTREE checkout test2 checkout-test2` test. Is there a bug there? Seems like it's hardlinking from a bare-user repo even without `-U`.